model/dao/ProjectDAO/PostgreSQLProjectDAO.php

```diff
@@ -610,9 +630,6 @@ public function partialUpdate(ProjectVO $projectVO, $update) {
         if ($update['areaId'])
         $sql = $sql . "areaid=" . DBPostgres::checkNull($projectVO->getAreaId()) . ", ";
 
-        if ($update['customerId'])
-            $sql = $sql . "customerid=" . DBPostgres::checkNull($projectVO->getCustomerId()) . ", ";
-
```

Comment:
Why did you remove this? The customer id field in the project may be updated, just like any of the other fields.